### PR TITLE
Add detailed install requirements for Linux

### PR DESCRIPTION
**Pull request by tjohnsonBCM**
Signed-off-by: Timothy Johnson <timothy.johnson@broadcom.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,6 +56,33 @@ Before you install and use the plug-in:
...
+
+    The second command will prompt for your password. Press Ctrl+D when you have finished typing it.
+    ```bash
+    export $(dbus-launch)
+    gnome-keyring-daemon --components=secrets --unlock
+    ```
+
+-   To automatically unlock the Gnome keyring at system boot, add the lines below to your `~/.bashrc` file.
+
+    Replace "root" with your system password.
```
Comment (tjohnsonBCM) [inline README.md:77]: Should there be a warning here the password is being stored in plain text, or can it be assumed that's obvious here?

Comment (awharn) [reply README.md:77]: I would suggest making it explicitly clear that this stores a password in plain text, and is not suggested. I still think we need to find a way to more reliably and securely make this available for headless Linux systems.

The problems are two-fold. Firstly, an X11 $DISPLAY must be available for dbus to start automatically. Secondly, the keyring needs to be unlocked once the dbus has been started. These may only affect the active connection, and may need to be run multiple times if more then one connection (such as using a remote VS Code Server via SSH or WSL).

**Comment by tjohnsonBCM** [comment]
> I would suggest making it explicitly clear that this stores a password in plain text, and is not suggested. I still think we need to find a way to more reliably and securely make this available for headless Linux systems.

@awharn @zFernand0 Agreed. I've updated the doc to state to suggest against storing passwords in plain text. Should we go ahead with merging this PR, or wait until a more reliable way is found? (There may be one described [here](https://wiki.archlinux.org/index.php/GNOME/Keyring#Using_the_keyring_outside_GNOME), but it requires more testing.)

**Comment by tjohnsonBCM** [comment]
Updated instructions for automatic keyring unlock at log on, so that they don't require entering system password in plain text.

The new instructions are based on Gnome keyring docs [here](https://wiki.archlinux.org/index.php/GNOME/Keyring#Using_the_keyring_outside_GNOME), and PAM config modifications from [here](https://github.com/AHumanFromCA/jenkins-npm-keytar). I have tested that they work in a CentOS VM.

File: README.md (hunk 1 above)
Comment (tjohnsonBCM) [reply README.md]: Updated the docs so they no longer require entering system password in plain text. The problem for remote VS Code users still exists, I suggest we create a separate issue for tracking it.

**Comment by tjohnsonBCM** [comment]
Thanks @awharn for testing on multiple headless Linux distributions 😄 Hopefully ready for merge now ༼ つ ◕_◕ ༽つ

**Pull request merged**